examples/cloudml-bee-health-detection/trainer/inputs.py

```diff
@@ -62,9 +61,9 @@ def _parse_csv(record):
     """
     column_defaults = [
         tf.constant([], tf.string),
-        tf.constant([], tf.int32),
+        tf.constant([], tf.string),
         tf.constant([], tf.int32)]
-    column_names = ['id', 'depth', TARGET_COLUMN]
+    column_names = ['file', 'subspecies', TARGET_COLUMN]
```

Comment:
What do you think about `img_file` instead of `file`?

Reply:
or `img_filename`

Reply:
I could but I was using the same names used in the original data. 
I'll change it